src/components/CheckoutTabs/CheckoutSummaryTab.tsx

```diff
@@ -0,0 +1,79 @@
+import { Button } from 'components/Buttons/Button';
+import { CheckoutFooter } from 'components/CheckoutFooter/CheckoutFooter';
+import { CheckoutSection } from 'components/CheckoutSection/CheckoutSection';
+import { OrderDetailsMinimal } from 'components/OrderDetails/OrderDetailsMinimal';
+import { CheckoutTabContext, CheckoutTabType } from 'context/CheckoutTabContext';
+import { useLocalOrder } from 'context/OrderContext';
+import { CreatedOrderDto } from 'dtos/Order/CreatedOrderDto';
+import { useOrder } from 'hooks/useOrder';
+import { useContext } from 'react';
+import { GBPFormat } from 'utilities/GBPFormat';
+import { SERVICE_CHARGE } from 'utilities/variables';
+
+export const CheckoutSummaryTab = (): React.ReactElement => {
+  const { createOrder, updateOrder } = useOrder();
+  const { localOrder } = useLocalOrder();
+  const { setCurrentTab } = useContext(CheckoutTabContext);
+
+  const orderExists = localOrder.lineItems?.length > 0;
+
+  const submitOrder = (): void => {
+    const orderLineItems = localOrder.lineItems.map((p) => ({ productId: p.product.id, quantity: p.quantity }));
+    const createdOrder: CreatedOrderDto = { lineItems: orderLineItems };
+
+    if (!localOrder.id) {
+      createOrder.mutate(createdOrder, {
+        onSuccess: () => {
+          setCurrentTab(CheckoutTabType.Confirmation);
+        },
+      });
+    } else {
+      updateOrder.mutate(
+        { id: localOrder.id, updatedOrder: createdOrder },
```

Comment:
I'm not sure we need the edit order here? We don't actually have an agreed idea of how a customer will go about editing their order - we need to think about that in the next milestone.

Reply:
i'll remove it for now